<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tap_input">
         <attribute name="title">
          <string>Input</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPlainTextEdit" name="input1">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <italic>false</italic>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="send_input"/>
            </item>
            <item>
             <widget class="QPushButton" name="button_send">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="button_clear_history">
              <property name="text">
               <string>Clear History</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_calib_led">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calibrate LED</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_calib_3V">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calibrate 3V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_calib_2V">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calibrate 2V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_calib_commit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Commit Calibration</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tap_catch">
         <attribute name="title">
          <string>Catch</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Capture first occurrence of: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="catch_phrase"/>
            </item>
            <item>
             <widget class="QPushButton" name="button_catch">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Catch</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="input_catch">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="button_refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="select_port"/>
      </item>
      <item>
       <widget class="QComboBox" name="select_baudrate"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_connect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
